### PR TITLE
WFCORE-3135 security-manager subsystem has wrong xsd schema

### PR DESCRIPTION
**Pull request by ctomc**
jira: https://issues.jboss.org/browse/WFCORE-3135
https://issues.jboss.org/browse/JBEAP-12532

### CODE REVIEW

**Comment by bstansberry** [comment]
This will need an acked Blocker JBEAP. This kind of mistake is a valid Blocker.

**Comment by ctomc** [comment]
created https://issues.jboss.org/browse/JBEAP-12532 for that

File: security-manager/src/main/resources/schema/wildfly-security-manager_3_0.xsd [hunk 1]
```diff
@@ -1,113 +0,0 @@
...
-        <xs:sequence>
-            <xs:element name="deployment-permissions" minOccurs="0" maxOccurs="1" type="deploymentPermissionsConfigType"/>
-        </xs:sequence>
-    </xs:complexType>
-
-    <xs:complexType name="deploymentPermissionsConfigType">
-        <xs:annotation>
-            <xs:documentation>
-                <![CDATA[
-                The deploymentPermissionConfigType specifies the elements and attributes that can be used to configure
```
Comment (rsvoboda) [inline security-manager/src/main/resources/schema/wildfly-security-manager_3_0.xsd]: pure removal of wildfly-security-manager_3_0.xsd is not the right thing to me.

`<xs:complexType name="deploymentPermissionsConfigType">` and `<xs:complexType name="permissionType">` has better `<xs:documentation>` in XSD version 3.0 comparing to 1.0

Thus I would suggest to also enhance XSD version 1.0 with the diff as part of this PR

Comment (bstansberry) [inline security-manager/src/main/resources/schema/wildfly-security-manager_3_0.xsd]: +1

Comment (ctomc) [inline security-manager/src/main/resources/schema/wildfly-security-manager_3_0.xsd]: update PR to use docs from 3.0 schema in 1.0

**Pull request merged**